pages/library/viewBook.js

```diff
@@ -85,9 +85,9 @@ export default function ViewBook({ library, currentBook }) {
       <BookPageContainer>
         <BookPageImage>
           <BookTitle>
-            <p>____________</p>
+            <div className="seperator">_________</div>
```

Comment:
I would style the divs instead of giving them content, but I think thats up to you.